components/password-reset-controller.js

```diff
@@ -0,0 +1,53 @@
+/*!
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+define([], function() {
+
+'use strict';
+
+function register(module) {
+  module.controller('brAuthnPasswordResetController', factory);
```

Comment:
If it will take too long to do, file an issue to do the component conversion and assign it to me.